.github/workflows/tools-test.yml

```diff
@@ -0,0 +1,31 @@
+name: tools-test
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
```

Comment:
Should we also test python 3.6 and 3.7 (currently the default python version on slower moving distributions like Debian and Cent-OS).

Reply:
We could but I fear that will add too many items in the list of checks: 1 for the tools-buildtest, 1 for the static-test and then  3 for the tools-test. + the Murdock and circleci ones.

Reply:
OK.